portfolio2023/index.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Yolis' Portfolio</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+    <body>
+        <div class="navbar">
```

Comment:
```suggestion
        <navbar class="navbar">
```

See my comment above about the benefit of using a `<navbar>` element (make sure to update the closing tag too)